clang_build/target.py

```diff
@@ -452,6 +460,11 @@ def __init__(self,
             if not target.__class__ is HeaderOnly:
                 self.link_command += ['-L', str(target.output_folder.resolve())]
 
+        if self.build_type == _BuildType.Debug:
```

Comment:
Why is there still this if-chaining here? Shouldn't that have been made simpler with a previous merge request?

Reply:
Indeed, this should be improved in the next iteration of cleaning up.